The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 08:30:30.501 UTC [43] LOG:  listening on IPv6 address "::1", port 5432
2025-02-14 08:30:30.501 UTC [43] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-14 08:30:30.506 UTC [43] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 08:30:30.521 UTC [44] LOG:  database system was shut down at 2025-02-14 08:30:30 UTC
2025-02-14 08:30:30.525 UTC [43] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE

CREATE ROLE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 08:30:30.893 UTC [43] LOG:  received fast shutdown request
2025-02-14 08:30:30.894 UTC [43] LOG:  aborting any active transactions
2025-02-14 08:30:30.895 UTC [43] LOG:  worker process: logical replication launcher (PID 50) exited with exit code 1
2025-02-14 08:30:30.895 UTC [45] LOG:  shutting down
2025-02-14 08:30:30.906 UTC [43] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 08:30:31.002 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 08:30:31.002 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 08:30:31.004 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 08:30:31.011 UTC [70] LOG:  database system was shut down at 2025-02-14 08:30:30 UTC
2025-02-14 08:30:31.014 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 08:30:36.696 UTC [85] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 08:30:36.696 UTC [85] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 08:30:37.076 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:30:37.076 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_7ba72d71-3a33-4b54-ab4a-f2ce17d6965f" WITH (FORCE)
2025-02-14 08:30:37.617 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:30:37.617 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_d7524a39-5d20-4dd1-b17b-9ec380627da8" WITH (FORCE)
2025-02-14 08:30:38.073 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:30:38.073 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_c98f4278-e269-4829-96e1-584fbb4ef7f0" WITH (FORCE)
2025-02-14 08:30:38.501 UTC [85] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:30:38.501 UTC [85] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_eea0650b-25f3-4e34-ac3f-12844e1a3060" WITH (FORCE)
